### PR TITLE
Dashboard Read and Delete

### PR DESCRIPTION
**Pull request by Gblur**

### CODE REVIEW

File: db/models/Map.js [hunk 1]
```diff
@@ -11,6 +11,15 @@ const MapSchema = new Schema({
 	edges: String,
 });
 
+const UserSchema = new Schema({
```
Comment (PReichetanz) [inline db/models/Map.js]: I guess it's fine, but we normally use a separate folder + file for different schemas.

File: pages/maps/[id]/index.js [hunk 2]
```diff
@@ -1,12 +1,48 @@
 import React, {useState} from "react";
 import Canvas from "../../../components/Canvas";
 import "reactflow/dist/style.css";
+import {shallow} from "zustand/shallow";
+import Button from "@mui/material/Button";
+import styled from "styled-components";
 // import Filter from "../../components/Canvas/FilterData";
```
Comment (PReichetanz) [inline pages/maps/[id]/index.js]:
```suggestion
```

File: pages/maps/[id]/index.js [hunk 3]
```diff
@@ -18,9 +54,32 @@ export default function MapDetailsPage() {
...
+					<h2>{map.name}</h2>
+					<p>{map.description}</p>
+				</div>
+				<div>
+					<Button
+						onClick={() => {
+							onNodeCreate(uuidv4());
+						}}>
+						Create Node
+					</Button>
```
Comment (PReichetanz) [inline pages/maps/[id]/index.js:67]:
```suggestion
					<Button
						onClick={() => {
							onNodeCreate(uuidv4());
						}}
						type="button">
						Create Node
					</Button>
```

**Review by PReichetanz — COMMENTED** [review]
Good work, only some minor issues to discuss 🚀

File: store/index.js [hunk 4]
```diff
@@ -73,6 +97,17 @@ const useStore = create((set, get) => {
...
+		deleteMap: (id) => {
+			handleDelete(id, get().fetchMaps);
+			// const filteredMaps = get().maps.filter((map) => {
+			// 	return map._id !== id;
+			// });
+
+			// console.log(filteredMaps);
+			// set({
+			// 	maps: filteredMaps,
+			// });
```
Comment (alinawinter) [inline store/index.js]: this can be deleted

**Review by alinawinter — APPROVED** [review]
This looks very complex! Cool work! I didn't see anything problematic (besides the comment) but I also didn't understand everyhting

**Pull request merged**